setup.py

```diff
@@ -17,11 +19,156 @@
 #  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 #  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 #  DEALINGS IN THE SOFTWARE.
+"""Molecule distribution package setuptools installer."""
 
 import setuptools
 
-setuptools.setup(
-    packages=setuptools.find_packages(),
-    pbr=True,
-    setup_requires=['pbr']
-)
+
+try:
```

Comment:
This all looks quite bleeding edge ... what are the risks here? Is this standard practice right now?

Reply:
It's a compatibility shim. Basically, I just emulate what newer setuptools have out of the box. So that ancient installations wouldn't explode. I do this in a few projects I maintain + `ansible-lint`.

Reply:
And new installations won't hit it at all. Because we provide a wheel which their pip will just unpack. So it only affects ancient installations in which pip doesn't know about preferring wheels

Reply:
OK, thanks!